base-template.sh

```diff
@@ -342,12 +343,12 @@ execute_opt_in_checks() {
             MESSAGE="Hook file changed"
         fi
 
-        echo "? $MESSAGE: $HOOK_PATH"
-
         if [ "$ACCEPT_CHANGES" = "a" ] || [ "$ACCEPT_CHANGES" = "A" ]; then
+            echo "? $MESSAGE: $HOOK_PATH"
             echo "  Already accepted"
         else
-            show_prompt ACCEPT_CHANGES "  Do you accept the changes?" "(Yes, all, no, disable)" "Y/a/n/d" "Yes" "All" "No" "Disable"
+            MESSAGE="$(printf "%s\n%s" "$MESSAGE: $HOOK_PATH" "  Do you accept the changes?")"
```

Comment:
What does this print?
Looks like printing on the terminal will output some messages more than it should now, right?